webodf/lib/ops/OdtDocument.js

```diff
@@ -712,7 +763,7 @@ ops.OdtDocument = function OdtDocument(odfCanvas) {
 
             if (cursorMoved) {
                 cursor.setSelectedRange(selectedRange, cursor.hasForwardSelection());
-                self.emit(ops.Document.signalCursorMoved, cursor);
+                eventNotifier.emit(ops.Document.signalCursorMoved, cursor);
```

Comment:
As per comment below, can rely on the event array being passed in. To allow external consumers outside the ops to call this function still, I would suggest the parameter take either an array or a boolean set to false. This means external consumers *must* specifically call this function with a value of "false" in order to execute (and it should throw if the value is undefined)